plugins/basic/README.md

```diff
@@ -1 +1,16 @@
-every plugin should have a readme file with info.
\ No newline at end of file
+# Maya Basic Toolkit workflow plugin
+
+This is a Shotgun Pipeline Toolkit plugin, 
+embedding the Shotgun Pipeline Toolkit and allowing
+you to easily run and deploy Toolkit Apps and Engines.
+
+- You can read more about Plugin development 
+  in our [Toolkit Developer Documentation](http://developer.shotgunsoftware.com/tk-core/bootstrap.html#developing-plugins).
+
+- The plugin needs to be built before it can be executed. You do this by 
+  executing the build tools found [here](https://github.com/shotgunsoftware/tk-core/blob/master/developer).
+
+- For more information about Toolkit, see http://support.shotgunsoftware.com/
+
+
```

Comment:
Who is the target audience of this file?
For developers, I would include more info about `tk-config-pluginbasic` and building the plugin.
For users, I would include the procedure to install it in Maya. This procedure is currently in `shotgun.mod`.